Sources/Tests/UnitTests/Calculus/SeriesTest.cs

```diff
@@ -19,6 +20,24 @@ public void TaylorDirect(string funcOverTRaw, string expectedRaw, int termCount)
             Assert.Equal(expected, actual);
         }
 
+        [Theory]
+        [InlineData("a t3 + b t2 + c t + d", "((d) + ((c) * (x))) + (((((2) * (b)) * ((x) ^ (2))) / ((2)!)) + ((((6) * (a)) * ((x) ^ (3))) / ((3)!)))", 4)]
+        [InlineData("(e^t)ln(1+s)", "((0) + ((y) + ((((x) * (y)) + -((y) ^ (2))) / ((2)!))))", 3)]
+        [InlineData("sin(t)+cos(s)+tan(u)", "((1) + ((x) + (z)))", 2)]
```

Comment:
To be honest... that's still too few tests. This is a big thing (multivariable taylor), it should be tested properly. Could you add tests for, like, 4 terms? Also, for single variable? For two? For four? For one term? Etc.